src/services/Authentication/signUpService.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+const signupHandler = (e, signupData, authDispatch, navigatorHelper) => {
+    e.preventDefault();
+    (async () => {
+        try {
+            const { data, status } = await axios.post("/api/auth/signup", {
+                ...signupData
+            })
+            if (status === 201) {
+                authDispatch({ type: "SIGN-UP", payload: data })
+                localStorage.setItem("authenticationToken", data.encodedToken);
+                navigatorHelper("/products")
+            }
+        } catch {
+            console.error("SIGN UP NOT POSSIBLE");
+        }
+    })()
+}
+export { signupHandler }
```

Comment:
Appreciate this part where you have created signIn, signUp and signOut as a service, this makes the code look a lot clean and also provides reusability. Great work there